packages/kubernetes-1.21/kubelet-config

```diff
@@ -74,7 +74,10 @@ systemReserved:
   {{@key}}: "{{this}}"
   {{~/each}}
 {{~/if}}
-cpuManagerPolicy: "static"
+cpuManagerPolicy: {{default "none" settings.kubernetes.cpu-manager-policy}}
```

Comment:
Same question as above.

```suggestion
cpuManagerPolicy: {{default "static" settings.kubernetes.cpu-manager-policy}}
```

Reply:
We heard from @bcressey out-of-band that he _does_ want to change the default back to `none` so I am resolving this issue and approving.